app/apollo/resolvers/cluster.js

```diff
@@ -203,7 +203,7 @@ const clusterResolvers = {
         //TODO: soft delete the resources for now. We need to have a background process to
         // clean up S3 contents based on deleted flag. 
         const deletedResources = await models.Resource.updateMany({ org_id, cluster_id }, 
-          {$set: { deleted: true }}, { upsert: false });
+          {$set: { deleted: true, expireAt: new Date() }}, { upsert: false });
```

Comment:
I think https://docs.mongodb.com/manual/reference/operator/update/currentDate/ might be a better option.  Just in case the database and server process are not in sync with time

Reply:
Modified date format(`new Date()`) to `$currentDate`. Please check